test/Altinn.App.Core.Tests/Internal/Process/ProcessEventHandlingTests.cs

```diff
@@ -533,17 +533,7 @@ public async Task UpdateProcessAndDispatchEvents_EndEvent_confirmation_instance_
             new InstanceEvent()
             {
                 EventType = InstanceEventType.process_EndEvent.ToString(),
-                ProcessInfo = new()
-                {
-                    CurrentTask = new()
-                    {
-                        ElementId = "Task_2",
-                        AltinnTaskType = "confirmation",
-                        Name = "Bekreft",
-                        Flow = 2
-                    },
-                    EndEvent = "EndEvent"
-                }
+                ProcessInfo = new() { CurrentTask = null, EndEvent = "EndEvent" }
```

Comment:
As far as I can tell EndEvent will not contain a current task, so this test was configured a bit wrong.